KeyboardSupport.xcodeproj/project.pbxproj

```diff
@@ -658,14 +714,14 @@
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = AJSV2L9F8Q;
+				DEVELOPMENT_TEAM = 2UAV38D2MU;
```

Comment:
Let's keep the development team and bundle IDs unchanged.

Reply:
🤦‍♂️ yes.

https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/170fbc27225cb347e97c16d4b37ada8486816355